.github/workflows/checks.yaml

```diff
@@ -0,0 +1,26 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+on:
+  workflow_call:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  check-style:
+    runs-on: ubuntu-latest
+    container:
+      image: rapidsai/ci-conda:latest
```

Comment:
Can we avoid using special images and run using `ubuntu-latest` *only*? I'm happy to merge this to get CI green for now and then work on it in a follow-up.

Reply:
I'm happy for us to do that - I don't quite know what's involved there but would be keen for us to do more standard things.

For background: the whole Numba-CUDA CI setup is derived from RAPIDS CI setups, which is why there are many RAPIDSisms in the setup here. Unfortunately I was not really knowledgeable enough to start afresh with a clean CI setup for Numba-CUDA (and leverage only the relevant parts of the RAPIDS setup) originally, and with the changes and additions over the past year there are quite a lot of anachronisms and oddities that have built up.